Oqtane.Server/Infrastructure/Localization/LocalizationSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using Oqtane.Shared;
+
+namespace Oqtane.Infrastructure.Localization
+{
+    public static class LocalizationSettings
```

Comment:
It would be nice if we avoid `static` and use options pattern, but the issue is we need a way to read a settings from within the extension methods